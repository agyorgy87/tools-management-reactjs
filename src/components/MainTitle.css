@import url('https://fonts.googleapis.com/css2?family=Jura&display=swap');

.mainTitle {
    margin-top: 20px;
    border: none;
    display: flex;
    background-color: #41414d;
    color: #fe8e22;
    font-size: 300%;
    font-weight: 600;
    font-family: 'Jura', sans-serif;
}
