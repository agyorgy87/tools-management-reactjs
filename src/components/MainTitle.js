import React, {useState} from 'react';
import '../css/MainTitle.css';

export default function MainTitle(props) {

    let tools = props.data;
    const[allData, setAllData] = useState(tools);

    const backToMain = (props) => {
        const backToAllData = allData
        props.setData(backToAllData);
    }
 
    return (
        <div>
            <button onClick={() => backToMain(props)}  className="main-title">Tool Shop</button>
        </div>
    )
}