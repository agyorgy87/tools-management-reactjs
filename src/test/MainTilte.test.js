import renderer from 'react-test-renderer';
import MainTitle from "../components/MainTitle.js";

test("test MainTitle", () => {
    let tree = renderer
    .create(<MainTitle data={[]} setData={() => {}}/>)
    .toJSON()
    expect(tree).toMatchSnapshot();
});