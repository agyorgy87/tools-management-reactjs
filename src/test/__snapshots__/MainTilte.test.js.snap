// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test MainTitle 1`] = `
<div>
  <button
    className="main-title"
    onClick={[Function]}
  >
    Tool Shop
  </button>
</div>
`;
